pom.xml

```diff
@@ -61,7 +67,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.13.4.2</version>
+            <version>2.15.1</version>
```

Comment:
This is the most recent `jackson-databind` dependency that supports the `logstash-logback-encoder`. The most recent versions at `2.15.X` does not have the `StreamConstraintsException` class